src/syscalls/main.rs

```diff
@@ -0,0 +1,305 @@
+use bcc::core::BPF;
+use bcc::BccError;
+use clap::{App, Arg, ArgMatches};
+
+use core::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::{mem, ptr, thread, time};
+
+mod syscall;
+
+// A tool for reporting syscall count and latency
+//
+// Based on: https://github.com/iovisor/bcc/blob/master/tools/syscount.py
+
+#[repr(C)]
+struct data_key_t {
+    count: u64,
+    total_ns: u64,
+}
+
+const MILLISECOND: u64 = 1_000 * MICROSECOND;
```

Comment:
This can be addressed in a follow-up - but we probably want to have a `src/common/..` that can have constants like this.